Classes/Queue/QueueInterface.php

```diff
@@ -102,13 +102,33 @@ public function finish($messageId);
     public function peek($limit = 1);
 
     /**
-     * Count ready messages in the queue
+     * Get a count of ready messages currently in the queue.
      *
-     * Get a count of messages currently in the queue.
+     * @return int The number of ready messages in the queue
+     * @deprecated Will be removed with the next major. Use countReady instead.
+     */
+    public function count(): int;
```

Comment:
If we change the interface we don't need to keep deprecated methods IMO as the implementations need to be adjusted anyways